ports/stm32/boards/OLIMEX_H407/mpconfigboard.mk

```diff
@@ -0,0 +1,11 @@
+MCU_SERIES = f4
+CMSIS_MCU = STM32F407xx
+AF_FILE = boards/stm32f405_af.csv
+LD_FILES = boards/stm32f405.ld boards/common_ifs.ld
+TEXT0_ADDR = 0x08000000
+TEXT1_ADDR = 0x08020000
+
+# MicroPython settings
+MICROPY_PY_LWIP = 1
+MICROPY_PY_USSL = 1
+MICROPY_SSL_MBEDTLS = 1
```

Comment:
since ETH is not enabled, these 3 settings can be removed

Reply:
Removed these lines and updated my branch.